blog/tests/models_test.py

```diff
@@ -0,0 +1,64 @@
+"""Tests for model's Blog app."""
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from blog.models import Post
+
+
+class PostModelTest(TestCase):
+    """Tests for Post model."""
+
+    @classmethod
+    def setUpTestData(cls):
+        """Set up non-modified objects used by all test methods."""
+        Post.objects.create(author=User.objects.create(username='bob'),
+                            title='Title', text='A few lines of text')
+
+    def test_author_label(self):
+        """Test for author label."""
+        post = Post.objects.get(id=1)
+        field_label = post._meta.get_field('author').verbose_name
+        self.assertEqual(field_label, 'author')
+
+    def test_title_label(self):
+        """Test for title label."""
+        post = Post.objects.get(id=1)
+        field_label = post._meta.get_field('title').verbose_name
+        self.assertEqual(field_label, 'title')
+
+    def test_text_label(self):
+        """Test for text label."""
+        post = Post.objects.get(id=1)
+        field_label = post._meta.get_field('text').verbose_name
+        self.assertEqual(field_label, 'text')
+
+    def test_created_date_label(self):
+        """Test for created_date label."""
+        post = Post.objects.get(id=1)
+        field_label = post._meta.get_field('created_date').verbose_name
+        self.assertEqual(field_label, 'created date')
+
+    def test_published_date_label(self):
+        """Test for published_date label."""
+        post = Post.objects.get(id=1)
+        field_label = post._meta.get_field('published_date').verbose_name
+        self.assertEqual(field_label, 'published date')
+
+    def test_author_name(self):
+        """Test for check name."""
+        post = Post.objects.get(id=1)
+        author_username = post.author.get_username()
+        self.assertEqual(author_username, 'bob')
+
+    def test_title_name(self):
+        """Test for check title."""
+        post = Post.objects.get(id=1)
+        post_title = post.title
+        self.assertEqual(post_title, 'Title')
+
+    def test_text_name(self):
+        """Test for check text."""
```

Comment:
What is "check text"?

Reply:
Removed this test. I think it was necessary.